uv.lock

```diff
@@ -327,17 +326,17 @@ wheels = [
 
 [[package]]
 name = "cyclopts"
-version = "3.24.0"
+version = "3.23.1"
```

Comment:
These are odd.. Can you ensure you have the latest? I just merged https://github.com/opendatahub-io/opendatahub-tests/pull/617